armada/handlers/wait_test.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+from armada import Armada
```

Comment:
organize your imports alphabetically - good practice is to put 

1 - std lib imports
2 - 3rd party imports
3 - project relevant imports